css/style.css

```diff
@@ -0,0 +1,143 @@
+/* Native Element overriding */
+ul {
+  list-style-type:none;
+  padding-left: 0px;
+}
+
+li{
+  display: inline-block;
+}
+
+body{
+  background: url("../images/background_guitar.png");
+  background-repeat: no-repeat;
+  background-position-y: -940px;
+  background-position-x: -712px;
+  background-size: 200%;
+}
+
+/* Class Styling */
+.col-xs-6{
+  margin: none;
+  padding: 0px 0px;
+}
+
```

Comment:
The class above and below are the same, target them both to stop it from being redundant.